FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDemoSpecs.m

```diff
@@ -282,40 +282,6 @@
             expect(view).toNot.haveValidSnapshotWithTolerance(0.24);
         });
     });
-    
-    it(@"matches view with 256 bytes aligned pointer", ^{
```

Comment:
I assume this test is a good one other than the crash? Open a new issue for this test crashing so we don't lose it?

Reply:
https://github.com/dblock/ios-snapshot-test-case-expecta/issues/50